Dshop/apps/users/api_views.py

```diff
@@ -45,4 +54,53 @@ def post(self, request):
             return Response(data, status=status.HTTP_200_OK)
         else:
             msg = {'detail': _('Unable to retrieve user auth token')}
-            return Response(msg, status=status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response(msg, status=status.HTTP_400_BAD_REQUEST)
+
+
+class LogoutView(GenericAPIView):
+    permission_classes = [AllowAny]
+    serializer_class = EmptySerializer
+    throttle_scope = 'logout'
+
+    def post(self, request):

```

Comment:
I would suggest using GET method here, not POST. You are not sending any data after all, right?

Reply:
https://www.reddit.com/r/learnprogramming/comments/go5l97/should_logout_in_websites_be_done_using_post/
> Best practice is to use POST with a CSRF token for logout. Otherwise you are leaving a potential path for an attacker to force logout